frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -47,7 +47,15 @@ export const EditZing = () => {
       .get(`${API_ROOT}${COURSE_API}/students/${courseId}`)
       .then((res: AxiosResponse<CourseStudentDataResponse>) => {
         setUnmatchedStudents(res.data.data.unmatched)
-        setStudentGroups(res.data.data.groups)
+        let groups = res.data.data.groups.map((group: Group) => {
```

Comment:
Is there any particular reason for using `let` here?

Reply:
Nope, good catch